models/classes/DataPolicyOrchestrator/Model/DataPolicyMessage.php

```
@@ -0,0 +1,82 @@
+{
+    private const BACKOFFICE_APP = 'backoffice';
+
+    public string $dataSubjectRawId;
+    public string $ownerApp;
+    public string $policyId;
+    public string $policyVersion;
+    public string $tenantId;
+
+    public function __construct(array $data)
+    {
+        foreach (get_class_vars(static::class) as $property => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)